packages/databricks-vscode/src/configuration/ConnectionManager.ts

```diff
@@ -250,8 +250,21 @@ export class ConnectionManager implements Disposable {
                 `Can't connect to the workspace`,
                 e
             );
-            if (e instanceof Error) {
-                window.showWarningMessage(
+            if (e instanceof ProcessError) {
+                window
+                    .showErrorMessage(
+                        `Can't connect to the workspace. Error executing Databricks CLI command.`,
+                        "Show Logs"
+                    )
+                    .then((choice) => {
+                        if (choice === "Show Logs") {
+                            commands.executeCommand(
+                                "databricks.bundle.showLogs"
+                            );
+                        }
+                    });
+            } else if (e instanceof Error) {
+                window.showErrorMessage(
                     `Can't connect to the workspace: "${e.message}"."`
                 );
```

Comment:
We can also offer to show the logs here, pointing to the full "Databricks Logs" output channel. 

Related: might be useful to create a wrapper for the `showErrorMessageWithLogs` logic, since this is very common pattern in the code-base

Reply:
I will take this in a separate PR?